### PR TITLE
Updated EqualityComparer<T>.Default example description

### PR DESCRIPTION
**Pull request by svick**
Companion PR to https://github.com/dotnet/samples/pull/65.

### CODE REVIEW

File: xml/System.Collections.Generic/EqualityComparer`1.xml [hunk 1]
```diff
@@ -130,15 +130,14 @@
    
   
 ## Examples  
- The following example creates two dictionary collections of type `Box`. Both dictionaries are created with the constructor that takes an <xref:System.Collections.Generic.EqualityComparer%601> as a parameter.  
+ The following example creates a custom collection of type `Box` and then searches for a box matching another box by calling the `FindFirst` method, twice.
```
Comment (pkulikov) [inline xml/System.Collections.Generic/EqualityComparer`1.xml]: First, @svick, not directly related to your changes, as it was also at the old text: can *...a custom collection of type `Box`...* be misread as if the collection has type `Box`. What about:
>The following example creates a collection that contains elements of the `Box` type...

Second, *...searches for a box matching another box...* -> *...searches it for a box matching given box...*

Comment (svick) [reply xml/System.Collections.Generic/EqualityComparer`1.xml]: Good suggestions, changed.

Though I didn't use "given box", I think that would make it sound as if this code was given the box by someone else.

Comment (pkulikov) [reply xml/System.Collections.Generic/EqualityComparer`1.xml]: Yes, I agree with you: *given box* is as we talk from the point of view of the `FindFirst` method, while the sentence is about the main program and *another box* fits perfect, especially with a new change that introduces the separate instance to search for.

**Comment by rpetrusha** [comment]
Closing and reopening so that license/cla check is reported.

**Comment by rpetrusha** [comment]
Closing and reopening to begin new build.

**Comment by rpetrusha** [comment]
Closing and reopening to begin new build with the latest code.

**Comment by mairaw** [comment]
Good to merge @rpetrusha?

**Pull request merged**